dev/tidb-in-kubernetes/get-started/deploy-tidb-from-kubernetes-kind.md

```diff
@@ -0,0 +1,300 @@
+---
+title: 使用 kind 在 Kubernetes 上部署 TiDB 集群
+summary: 使用 kind 在 Kubernetes 上部署 TiDB 集群
+category: how-to
+---
+
+# 使用 kind 在 Kubernetes 上部署 TiDB 集群
+
+本文介绍了如何在个人电脑 （Linux 或 MacOS) 上采用 [kind](https://kind.sigs.k8s.io/) 方式在 Kubernetes 上部署 [TiDB Operator](https://github.com/pingcap/tidb-operator) 和 TiDB 集群。
+
+kind 通过 Docker 容器模拟出一个单点的 Kubernetes 集群。 kind 的设计初衷是为了在本地进行 Kubernetes 集群的一致性测试，这意味着你可以使用 kind 模拟出你想要的 Kubernetes 版本集群，你可以在 [Docker hub](https://hub.docker.com/r/kindest/node/tags) 中找到你想要部署的 Kubernetes 版本。
+
+> **警告：**
+>
+> 对于生产环境，不要使用此方式进行部署。
+
+## 环境准备
+
+部署前，请确认软件、资源等满足如下需求：
+
+- 资源需求 CPU 2+，Memory 4G+
+
+    > **注意：**
+    >
+    > 对于 macOS 系统，需要给 Docker 分配 2+ CPU 和 4G+ Memory。详情请参考 [Mac 上配置 Docker](https://docs.docker.com/docker-for-mac/#advanced)。
+
+- [Docker](https://docs.docker.com/install/)：>= 17.03
+- [Helm Client](https://github.com/helm/helm/blob/master/docs/install.md#installing-the-helm-client): 版本 >= 2.9.0 并且 < 3.0.0
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl): 至少 1.10，建议 1.13 或更高版本
+
+    > **注意：**
+    >
+    > 不同版本 `kubectl` 输出可能略有不同。
+
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/): 版本 >= 0.5.1
+
+## 第 1 步: 通过 kind 部署 Kubernetes 集群
+
+首先，请确认 Docker 进程正常运行。然后你可以通过 kind 命令快速启动一个本地的单点 Kubernetes 集群( kind 0.5.1 默认为 1.15.3 版本）。
+
+创建集群,这里我们先手动指定为 1.14.3 版本:
```

Comment:
Why use 1.14.3? If we recommend users to try 1.14, maybe the latest release of 1.14, e.g. 1.14.6?